packages/services/f-globalisation/src/mixins/globalisation.mixin.vue

```diff
@@ -1,5 +1,6 @@
 <script>
 export default {
+    name: 'GlobalisationMixin',
```

Comment:
Is this a new rule that means we have to add a name to mixins?

Reply:
Nope – it's basically my quick fix for the lining complaint of having one-word component names. I didn't want to update all the files and references (as there are quite a lot of errors across the repo like this) so I just added names into the exports.